app/views/shared/vote/_edit.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= form_for(vote, url: vote_path(vote), action: "update") do |f| %>
+  <%= f.button type: 'submit', class: "my-button clickable", id: vote_type do %>
+    <%= tag.i class: "fa fa-chevron-circle-#{vote_type}" %>
```

Comment:
what is `tag` here? I've never seen this before.

Reply:
tag is a rails helper for HTML tags - http://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-tag

I needed something to support the `<i>` that font awesome uses